apiproject.py

```diff
@@ -0,0 +1,106 @@
+# of this software and associated documentation files (the "Software"), to 
+# deal in the Software without restriction, including without limitation the 
+# rights to use, copy, modify, merge, publish, distribute, sublicense, and/or 
+# sell copies of the Software, and to permit persons to whom the Software is 
+# furnished to do so, subject to the following conditions:
+
+# The above copyright notice and this permission notice shall be included in 
+# all copies or substantial portions of the Software.
+
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR 
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, 
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE 
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER 
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING 
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER 
+# DEALINGS IN THE SOFTWARE.
+
+# IMPORTS #
+import sys
+import pilotUtils
+
+import urllib3
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning) #Disables warning for not verifying local user. Production will use UUID, dev does not
+
+# if len(sys.argv) != 4:
+#   print('ERROR: usage {} <customer-ID> <username> <password>'.format(sys.argv[0]))
+#   sys.exit(1)
+
+HOST = 'aptest.colo.seagate.com:32324'
+CUSTOMER_UUID = '' #'624f6357f47dcc0012126300' #sys.argv[1] # specific instance UUID
+USERNAME = 'admin@seagate.com' #sys.argv[2]
+PASSWORD = '389LyvePilot' #sys.argv[3]
+
+# Define API URLs
+VOLUMES_URL = 'https://{}/udx/v1/volumes'.format(HOST)#, CUSTOMER_UUID)
+DEVICES_URL = 'https://{}/udx/v1/devices'.format(HOST)#, CUSTOMER_UUID)
+IMPORT_URL = 'https://{}/udx/v1/import'.format(HOST)#, CUSTOMER_UUID)
+ACTIVITY_URL = 'https://{}/udx/v1/activity'.format(HOST)#, CUSTOMER_UUID)
+
+def getActivity(jwt, filterParams):
+  return pilotUtils.getHelper(jwt, ACTIVITY_URL, filterParams)
+
+def getDevices(jwt):
+  return pilotUtils.getHelper(jwt, DEVICES_URL, {})  
+
+def getVolumes(jwt, filterParams):
+  return pilotUtils.getHelper(jwt, VOLUMES_URL, filterParams) 
+
+def importData(jwt, sourceUri, destinationUri, destDeviceId, filterStr='*'):
+  reqData = {
+    'sourceUri': str(sourceUri),
+    'destinationUri': str(destinationUri),
+    'filter': filterStr,
+    'deviceId': str(destDeviceId),
+    'orchestrationMode': 'STANDARD_SECURITY'
+  }
+  return pilotUtils.postHelper(jwt, IMPORT_URL, reqData)
+
+# Get login token
+pilotUtils.printBold('\n----- LOGIN -----')
+token = pilotUtils.login(HOST, USERNAME, PASSWORD)['token']
+# print(token)
+
+# pilotUtils.printBold('\n----- DEVICES -----')
+# devices = getDevices(token)
+# deviceID = getDevices(token)['data'][0]['_id'];
+# print(deviceID);
+# pilotUtils.printJson(devices);
+
+pilotUtils.printBold('\n----- USERS -----')
+responseData = pilotUtils.getUsers(HOST, token)
+for i in responseData['data']:
+  print(i['name'])

```

Comment:
perhaps more informative print statements, example:
`print('User name: %s\n' % i['name'])`

Reply:
I also had the thought of having the function do that behind the scenes but I thought it might be beneficial for the user to see that the output was parsed